sdk/java/src/main/java/io/juicefs/JuiceFileSystemImpl.java

```diff
@@ -581,7 +581,7 @@ public static Libjfs loadLibrary() throws IOException {
     LibraryLoader<Libjfs> libjfsLibraryLoader = LibraryLoader.create(Libjfs.class);
     libjfsLibraryLoader.failImmediately();
 
-    int soVer = 8;
+    int soVer = 7;
```

Comment:
9?

Reply:
9 is used for 1.1-beta1. As discussed with tommy, we'd like to always use 7 for the main branch, and +1 every time a new version is released.